src/filter.ts

```diff
@@ -601,6 +601,7 @@ interface Shortcuts<U extends Update> {
         : [U["callback_query"]] extends [object]
             ? U["callback_query"]["message"]
         : undefined;
+    txt: [Shortcuts<U>["msg"]] extends [object] ? string : undefined;
```

Comment:
Not all messages have text. You need to map this to the actual runtime behaviour.

Reply:
Now its fine, I think(?)

Reply:
I believe so, but I haven't tested it myself